back/db/migrate/20241011816395_add_autoshare_attribute_to_phase_context.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class AddAutoshareAttributeToPhaseContext < ActiveRecord::Migration[7.0]
+  def change
+    %i[phases].each do |tablename|
+      add_column tablename, :autoshare_results_enabled, :boolean, default: true, null: false
+    end
```

Comment:
Don't really need to loop through an array of one!

```suggestion
      add_column :phases, :autoshare_results_enabled, :boolean, default: true, null: false
```